processor/src/test/java/org/mapstruct/ap/testutil/compilation/annotation/ExpectedNote.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.testutil.compilation.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * An expected {@link javax.tools.Diagnostic.Kind#NOTE}.
+ *
+ * @author Sjaak Derksen
+ */
+
+@Repeatable(ExpectedNote.ExpectedNotes.class)
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface ExpectedNote {
+
+    String value();
+
+    /**
+     * The notes in the order they are expected
+     */
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target(ElementType.METHOD)
+    public @interface ExpectedNotes {
+
+        /**
+         *  Regexp voor the value.
```

Comment:
In Germany we often say "denglisch" in case someone mixes English and German ("deutsch"). But maybe we can also say this for mixing Dutch and English? 🤔 

```suggestion
         *  Regexp for the expected note.
```

Reply:
ha! well spotted @chris922 .. its the same here..